lambda-buffers-frontend/lambda-buffers-frontend.cabal

```
@@ -0,0 +1,135 @@
+cabal-version: 3.0
+name:          lambda-buffers-frontend
+version:       0.1.0.0
+synopsis:      Lambda Buffers Frontend
+
+-- license:
+
+author:        MLabs LTD
+maintainer:    info@mlabs.city
+
+-- A copyright notice.
+-- copyright:
+-- category:
```

What's the Licensing on this project? 
```suggestion
-- :fixme:
-- A copyright notice.
-- copyright:
-- category:
```

---

Opened https://github.com/mlabs-haskell/lambda-buffers/issues/14